src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
move font-family to `html`

Reply:
still not fixed

Reply:
@etojeDenys 
 I don't understand how I should move to html if the fonts seem to be already in html

Reply:
![image](https://user-images.githubusercontent.com/125212635/225386626-4a687b39-48aa-4f57-bb05-538216d4c1b1.png)

Reply:
you need to move it into html selector in CSS file